src/components/ListItem.jsx

```diff
@@ -1,13 +1,25 @@
 import { useState, useEffect } from 'react';
 import './ListItem.css';
 import { updateItem } from '../api/firebase';
+import { deleteItem } from '../api/firebase';
 
 export function ListItem({ item, listId }) {
 	const [checked, setChecked] = useState(false);
+	const [deleteError, setDeleteError] = useState('');
 
 	const checkItem = (e) => {
 		updateItem(listId, item);
 	};
+	async function handleDelete() {
+		if (window.confirm('Are you sure you want to delete this item?')) {
```

Comment:
I know that you made this decision for accessibility issues versus using an alert box. I'll be keeping this in mind for the future!